src/Containers/ProductListings/Containers/ProductListingTable.tsx

```
@@ -58,12 +57,35 @@ export const ProductListingTable = observer(
 		);
 
+		const handleStickyScroll = (event: any) => {
+			if (mainScrollRef.current && !isMainScrolling.current) {
+				mainScrollRef.current.scrollTo({
+					y: event.nativeEvent.contentOffset.y,
+					animated: false,
+				});
+			}
+		};
+
+		const handleMainScroll = (event: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)